src/scripts/install-nvm.sh

```diff
@@ -10,6 +10,8 @@ fi
 
 if [ "$NODE_PARAM_LTS" = "1" ]; then
     nvm install --lts
+    nvm alias default lts
+    nvm use --lts
```

Comment:
It seems like this could be simplified by removing this line, and moving line 18 outside of the conditional?

Reply:
Thanks for the recommendation, how about now?

Reply:
Looks good to me!